apps/src/sites/studio/pages/congrats/index.js

```diff
@@ -6,9 +6,30 @@ import Congrats from '@cdo/apps/templates/Congrats';
 
 $(document).ready(function () {
   const isRtl = isRtlFromDOM();
+  const script = document.querySelector('script[data-congrats]');
+  const congratsData = JSON.parse(script.dataset.congrats);
+
+  let completedTutorialType = '';
+  switch (congratsData.completedTutorialType) {
+    // Minecraft designer or Minecraft adventurer
+    case 'minecraft' || 'mc':
+        completedTutorialType = 'pre2017Minecraft';
+        break;
+    // 2017 Minecraft
```

Comment:
I haven't tested new Minecraft and applab yet, but the old minecrafts and other tutorials thread the correct info through to the Congrats component.